release-blocking-jobs.md

```diff
@@ -1,83 +1,101 @@
+teams (such as the Release Engineering) or as "parking" for jobs that need to
+be fixed.
 
-The ideal release-blocking job meets the following criteria:
+If you want to add a release-blocking job, please add them to the appropriate
+Informing dashboard first. The release team can later decide if the test should
+be moved to the Blocking dashboard.
 
-- It provisions clusters via open-source software instead of a hosted service
-<!-- TODO(spiffxp):
-  - should any allowances be made for conformance?
--->
-- It finishes a run in 120 minutes or less
+Additional documentation may be found in the [CI Signal Role Handbook](./release-team/role-handbooks/ci-signal/README.md).
+
+## Release-Blocking Dashboard
+
+Jobs on the Blocking dashboard for a particular release should block that
+release if they do not have at least three successive "green" runs at go/nogo
+decision time.  In some cases, the CI Signal team may authorize release with a
+job still failing for thoroughly investigated reasons, but the general
+expectation is "jobs red, do not release."
+
+As such, criteria for what jobs are on Blocking are rather stringent, in order
+to prevent aborting a release unnecessarily.  Note that not all current jobs
+meet the below criteria completely due to technical debt.  Blocking jobs must:
+
+- Have a documented reason for its inclusion in the release-blocking suite
 <!-- TODO(spiffxp):
-  - this should be a percentile, not a max
+  - represented in description? link to an issue where discussion was had?
 -->
-- It runs at least every 3 hours
-- Is capable of passing 3 times in a row against the same commit
-- Is owned by a SIG that is responsive to addressing failures
-  - Represented by a configured testgrid alert email (enforced via presubmit)
-- It passes 75% of all of its runs in the past week (regardless of underlying
-  commit(s)) and fails for no more than 10 runs in a row
-  - In the case of failures, there must be an issue in kubernetes/kubernetes
-    detailing that there is at least one person from the owning SIG working to
-    resolve the issue. Said issue must be acknowledged within a short period
-    (i.e., a week) and resolved within a short period being acknowledged by the
-    owning SIG (i.e., a week as well).
-- Has a documented reason for its inclusion in the release-blocking suite
+- Provision clusters via open-source software instead of a hosted service
 <!-- TODO(spiffxp):
-  - represented in description? link to an issue where discussion was had?
+  - should any allowances be made for conformance?
 -->
-
-The reasoning for the above is that as we get to the end of a release cycle,
+- Have 75% of runs, or more, finishing in 120 minutes or less
+- Run at least every 3 hours
+- Be able to pass 3 times in a row against the same commit
+- Be Owned by a SIG, WG, or other team that is responsive to addressing failures, and whose alert email is configured in the job.
```

Comment:
Are you saying the goal **isn't** for 1.x-blocking jobs to meet the same criteria, whether or not they do?

Reply:
I think I was asking if this is still the goal, if so I think we should be explicit about what's in place vs. aspirational

Reply:
Only subprojects own code and by, extension, the SIG. Let's remove mention of WGs here.
And yes, it would be good to delineate between aspirational vs what's in place today.